frontend/src/components/UI/TabPanels/Tabs.vue

```diff
@@ -117,30 +117,18 @@
 									class="q-gutter-md"
 									filled
 									label="Git provider URL"
-									lazy-rules
-									:rules="[
```

Comment:
Those changes is added in the commit [add git fields to product creation form](https://github.com/ditrit/leto/pull/187/commits/8cc2124bb1fab26ccab04278f0c1366d6b72491d)
I prefere that you squash the commit `remove non necessary input rules` in `add git fields to product creation form`